README.md

```diff
@@ -0,0 +1,68 @@
+# OFirehose
+
+Firehose for the federated social web
+
+## Installation
+
+### Prerequisites
+
+You'll need four things to get started:
+
+* node.js 4.x or higher
+* npm 1.1.0 or higher
+
+### Local install
+
+You need clone the git repository:
+
+    git clone https://github.com/e14n/ofirehose.git
+
+And install the dependencies using `npm`:
+
+    cd ofirehose
+    npm install
+
+### Database setup
+
+OFirehose uses [databank][databank] Any databank driver should work. 
+`couchbase`, `mongodb` and `redis` are probably the best
+bets for production servers, but the `disk` or even `memory` drivers
+can work for testing.
+
+If you're confused, just use the *MongoDB* one, [databank-mongodb](https://www.npmjs.com/package/databank-mongodb).
+
+You can find other drivers like so:
+
+    npm search databank
+
+You can install in the `ofirehose` directory as a normal dependency.
+For example, the databank driver for MongoDB:
+
+    npm install databank-mongodb
+
+Note that you also need to install and configure your database server. 
+example in Debian derivatives: 
+
+    apt-get install mongodb
+
+## Configuration
+
+OFirehose require a configuration file located in `/etc/ofirehose.json`, `~/.ofirehose.json` or `./ofirehose.json`, any of the three options would be fine.
+
+### Available parameters
+
+* **driver** The [databank][databank] driver you're using. Defaults to `memory`, not recommended for production.
+* **params** Databank driver params; see the databank driver README for details on what to put here.
+* **key** If you're using SSL, the path to the server key, like
+   "/etc/ssl/private/myserver.key". Defaults `null`.
+* **cert** If you're using SSL, the path to the server cert, like
+   "/etc/ssl/private/myserver.crt". Defaults `null`.
+* **address** The address to listen on. Defaults server hostname, Use this if you've got some kind of load-balancer or NAS or whatever and your local IP doesn't map to the IP of the hostname.
+* **port** Port to listen on. Defaults to `443` when *key* is set or `80` otherwise, Change this if you've got some kind of load-balancer.
+   
+## License
+
+OFirehose is licensed under the Apache License Version 2.0, See the [LICENSE][LICENSE] file for the full license text.
```

Comment:
Should probably put a copyright notice in this section, something like:

```markdown
Copyright 2012-2014 E14N https://e14n.com/
Copyright 2017 AJ Jordan <alex@strugee.net>, Camilo QS <email@ifyouwantitlisted.tld>
```

Reply:
Also ProTip:tm: - you can actually write `[LICENSE][]` and it'll still work!

Reply:
Or `Copyright 2012-2014 E14N https://e14n.com/ and contributors` ?

Reply:
Idk. I think we're not supposed to since there's a gap of a couple years (2014-2017) where there was no copyrighted work generated. Also I think it's nice to list individual contributors, especially here when the list is pretty small.

Reply:
Sounds good to me